CONTRIBUTING.md

```diff
@@ -161,6 +162,34 @@ This project uses Prettier for code formatting. You can run `make format` to for
       - `Description` — the description of the prop, e.g. `Title of the accordion`
    3. The props MUST be sorted alphabetically by their name.
 
+## Testing
+
+Each package contains a script called `test`.
+Using this you can test entire package and verify that all parts of the package are in good shape and all rules are met.
+
+Testing script includes:
+
+- linting using [ESlint][eslint]
+- checking formatting using [Prettier][prettier]
+- checking types using [Typescript][typescript] compiler
+- running unit test using [Jest][jest]
+
+### Unit testing
+
+You can run unit testing via `test:unit` scripts.
+For all available scripts see the package's `package.json` file.
+
+### Visual regression testing
```

Comment:
As this PR only adds basic functionality and we are not actually able to use this for any testing, we should mention this here. When we figure out how we want to use it for testing, we can extend these docs.

Reply:
Ok, added another section.